cypress/support/commands.ts

```
@@ -2,35 +2,163 @@
 
 import { jwtDecode } from "jwt-decode";
 
+let userHeaderFixture: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)